### PR TITLE
feat(material-experimental/mdc-slider): add skeleton code for MatSlid…

### PR DESCRIPTION
**Pull request by wagnermaciel**
…erThumb

* created slider-thumb.ts
* created MatSliderThumb directive for the mdc-slider input

### CODE REVIEW

File: src/material-experimental/mdc-slider/slider-thumb.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import {NumberInput} from '@angular/cdk/coercion';
+import {DOCUMENT} from '@angular/common';
+import {Directive, ElementRef, EventEmitter, Inject, Input, Output} from '@angular/core';
+import {Thumb} from '@material/slider';
+
+/**
+ * The native input used by the MatSlider.
+ */
+@Directive({
+  selector: '[mat-slider-thumb]',
```
Comment (crisbeto) [inline src/material-experimental/mdc-slider/slider-thumb.ts:18]: Should the selector be limited to `input` elements?

File: src/material-experimental/mdc-slider/slider-thumb.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  /** Indicates which slider thumb this input corresponds to. */
+  thumb: Thumb;
+
+  constructor(
+    @Inject(DOCUMENT) private readonly _document: Document,
+    private readonly _elementRef: ElementRef<HTMLInputElement>,
+    ) {}
+
+  /** Returns the hosts native HTML element. */
+  getHostEl(): HTMLInputElement {
```
Comment (crisbeto) [inline src/material-experimental/mdc-slider/slider-thumb.ts:70]: Are we expecting this to be used as a public API for only for the slider itself? If it's the latter, it should be prefixed with an underscore. Same goes for `isFocused`. Also we should avoid abbreviating method names so this should be called `getHostElement`.

**Comment by wagnermaciel** [comment]
@crisbeto I made those changes 👍🏽

**Review by crisbeto — APPROVED** [review]
LGTM

**Pull request merged**